compute_sdk/globus_compute_sdk/sdk/shell_function.py

```diff
@@ -36,13 +36,20 @@ def __init__(
         self.returncode = returncode
         self.exception_name = exception_name
 
-    def __str__(self):
-        return f"Command {self.cmd} returned with exit status: {self.returncode}"
-
-    def __repr__(self):
-        return (
-            f"{self.__class__.__name__}(returncode={self.returncode}, cmd={self.cmd})"
+    def __str__(self) -> str:
+        # Format: Command { [cmd] ;}  -> Specific request: copy-paste-able
+        return f"  Command: {{ {self.cmd} ;}}\nExit code: {self.returncode}"
```

Comment:
Nice! This grouping makes sense here!
Any reason why there's some whitespace in front of `Command:`?

Reply:
Yes.  To line up with the colon of Exit code on the following line:
```
  Command: { some command here ;}
Exit code: 42
```

Reply:
Ah, I see.